pkg/controller/infrastructure/actuator.go

```diff
@@ -59,7 +59,7 @@ var StatusTypeMeta = func() metav1.TypeMeta {
 }()
 
 // NewActuator instantiates an actuator with the default dependencies.
-func NewActuator(machineImageOwnerSecretRef *corev1.SecretReference, whitelistedImageIDs []string) infrastructure.Actuator {
+func NewActuator(machineImageOwnerSecretRef *corev1.SecretReference, whitelistedImageIDs []string, enableSshKeyRotation bool) infrastructure.Actuator {
```

Comment:
Unused param `enableSshKeyRotation`.

```suggestion
func NewActuator(machineImageOwnerSecretRef *corev1.SecretReference, whitelistedImageIDs []string) infrastructure.Actuator {
```